### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by karwan01**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,6 @@
   "extends": "@vue/tsconfig/tsconfig.web.json",
   "include": ["env.d.ts", "src/**/*", "src/**/*.vue"],
   "compilerOptions": {
-    "ignoreDeprecations": "5.0",
```
Comment (Ge6ben) [inline tsconfig.json:5]: Hi @karwan01  Good job Can I know Why u remove it?!

Comment (karwan01) [reply tsconfig.json:5]: Hi @Ge6ben  the reason was this line was ignoring the deprecation's for version ts version 5

**Review by Ge6ben — APPROVED** [review]
Good job u can merge

**Comment by karwan01** [comment]
> Good job u can merge

**Pull request merged**